src/commands/mock.ts

```diff
@@ -47,5 +47,31 @@ export const Mock = {
         }
         mockTargets.add(target.id);
         await interaction.reply(`Sure thing, I will begin mocking ${target}!`);
+    },
+
+    Effect: async (message: Message): Promise<void> => {
+
+        let shouldBeLower = true;
+        let length = message.content.length;
+        let userMessage = message.content
+        let mockMessageArray = [...userMessage];
+
+        for (let i = 0; i < length; i++) {
```

Comment:
This is a very interesting approach you have. I think it would be easier to use string concatenation, something like
```ts
let mockMessage = "";
for (let i = 0; i < length; i++) {
    if (...) {
        mockMessage += userMessage[i];
    }
}
```